blog/secadvisor.md

```diff
@@ -0,0 +1,290 @@
+---
+title: How to use Security Advisor with Skydive
+layout: blog-post
+author: Kalman Meth
+date: 31/07/2019
+---
+
+
+The Security Advisor filters the flow data obtained from Skydive, performs a data transformation, and saves the information to an object store in compressed JSON format.
+This data may then be used to perform various kinds of analyses for security, accounting, or other purposes.
+
+Instructions to developers to deploy and run the Secruity Advisor can be found at https://github.com/skydive-project/skydive/tree/master/contrib/pipelines/secadvisor.
+We present here extended instructions for users to deploy Skydive and the Security Advisor.
+
+We use the term "pipeline" to describe the sequence of operations performed on the data: classify, filter, encode, store.
+In the future, we plan to have additional pipelines performing different kinds of operations.
+
+## Deploy Skydive
+
+You can either download a statically linked version of Skydive or you can build it on your own.
+
+To download a pre-compiled statically linked version of Skydive, run the following command.
+
+```
+curl -Lo skydive https://github.com/skydive-project/skydive-binaries/raw/jenkins-builds/skydive-latest && chmod +x skydive && sudo mv skydive /usr/local/skydive/
+```
+
+To build your own version of Skydive, follow the instructions on http://skydive.network/documentation/build to install prerequisites and prepare your machine to build the Skydive code.
+Enter the Skydive root directory and build the code.
+
+```
+cd $GOPATH/src/github.com/skydive-project/skydive
+make
+cp etc/skydive.yml.default /etc/skydive/skydive.yml
+// adjust settings in skydive.yml, if desired
```

Comment:
use a bash comment `#` not a go/c/c++ comment `\\`

Reply:
OK